### DISCUSSION TITLE
请问页底部有空白行该怎么处理

### DISCUSSION THREAD

**Discussion by Jamie-obj** [note286/xduts#122] [Q&A] [answered]
如图所示，上一页的空白行明显能够放的下下一页的标题以及文字，但这个（9）就是不上去，这种情况该怎么办
![image](https://user-images.githubusercontent.com/65389500/234747571-d4dff841-89b2-4130-8be3-841d9cb47ccb.png)

**Comment by note286** [comment]
放不下的，标题不能单独在页尾。

**Reply by Jamie-obj** [reply]
> 放不下的，标题不能单独在页尾。

那这种空白只能通过加减文字避免吗

**Reply by sikouhjw** [reply]
> 那这种空白只能通过加减文字避免吗

不需要避免，这不是错误也不是警告

**Reply by note286** [reply, marked as answer]
一般来说是的，或者通过设置`style/page-vertical-align`，但是有的垂直间距会被拉大，估计你更不喜欢。

**Reply by Jamie-obj** [reply]
> > 那这种空白只能通过加减文字避免吗
> 
> 不需要避免，这不是错误也不是警告

但是毕业论文要求避免这种底部留白

**Reply by note286** [reply]
那你设置`分散对齐`好了。

**Reply by Jamie-obj** [reply]
> 那你设置`分散对齐`好了。

好的，感谢解答
